src/qt/optionsmodel.cpp

```diff
@@ -90,6 +90,19 @@ void OptionsModel::Init()
         settings.setValue("nThreadsScriptVerif", DEFAULT_SCRIPTCHECK_THREADS);
     if (!SoftSetArg("-par", settings.value("nThreadsScriptVerif").toString().toStdString()))
         addOverriddenOption("-par");
+    if (!settings.contains("bPrune"))
+        settings.setValue("bPrune", false);
+    if (!settings.contains("nPruneSize"))
+        settings.setValue("nPruneSize", 200.0); // GB
+    if (settings.value("bPrune").toBool()) {
+        int mb = static_cast<int>(settings.value("nPruneSize").toDouble() * 1000);
```

Comment:
Can you check this again. I think `-prune` is one of the options in MiB, so I'd prefer to have consistency between bitcoind and bitcoin-qt.